src/components/media-loader.js

```diff
@@ -151,9 +151,11 @@ AFRAME.registerComponent("media-loader", {
           { once: true }
         );
         this.el.addEventListener("model-error", this.onError, { once: true });
+
+        const resolved = await resolveMedia(src);
```

Comment:
This url has already been resolved above on line 98, you should just be able to use `raw` and `contentType` already in this scope.